OlinkAnalyze/vignettes/bridging_E3072toEHT.Rmd

```diff
@@ -0,0 +1,682 @@
+considers one anchor point (the median/50% quantile). Instead, quantile
+smoothing (QS) using multiple anchor points (5%, 10%, 25%, 50%, 75%, 90%
+and 95% quantiles) is favored to map the Explore 3072 data to the
+Explore HT distribution. The normalization using QS uses bridging samples
+to perform the following steps:
+
+1.  Each data point of the samples from Explore 3072 is mapped to the
+    equivalent space in Explore HT using an empirical cumulative
+    distribution function. An empirical cumulative distribution function
+    is a probability model which uses the observed data, in this case
+    the NPX values of the bridging samples for an assay, to create a step
+    function which interpolates linearly between the available data
+    points.
+
+2.  The empirical distribution function is used to map the data points
+    from Explore 3072 to the Explore HT space using the specified
+    quantiles. At this point all data points from the bridging samples
+    have NPX values that are normalized to the data points in Explore
+    HT.
+
+3.  To normalize the remaining data a spline regression model is
+    constructed using the sorted Explore 3072 data and the mapped
+    Explore 3072 data along with the anchor points of the spline
+    function. A spline regression model divides a data set at the
+    quantiles and uses the quantile as an anchor point or knot. Then a
+    model is generated to fit the points between each anchor point.
+
+4.  The spline regression model is then used to predict all the data
+    points from Explore 3072 to Explore HT. The spline regression model
+    results in a combination of linear regression models within
+    intervals. The Explore 3072 NPX values are input as the x value
+    within the corresponding interval, which results in a y value
+    equivalent to the Explore HT NPX value.
+
+### Function Output
+
+The output from `olink_normalization_bridge_product()` function is a
+dataframe with concatenated data from the two products and additional
+columns including adjusted NPX values, bridging recommendations, mapping
+information, and project names. The adjusted NPX values are notated in
+the columns `MedianAdjustedNPX` and `QSNormalizedNPX`. For each assay a
+recommendation is listed in the `BridgingRecommendation` column and
+lists what method, if any should be used for that assay. Additional
+columns including `OlinkID_HT` and `OlinkID_3072` map the assays across
+products and the `Project` column lists the name of the project based on
+the `exploreht_name` and `explore3072_name` arguments. The resulting
+data set will contain the newly bridged Explore 3072 data set. The
+reference Explore HT data will be concatenated to the Explore 3072 data.
+As the reference data is not altered during normalization, the
+normalized NPX values in the Explore HT data will be the same as the
+values in the NPX column which contains the non-normalized data.
+
```

Comment:
Should there be another column in the example table below called OlinkID_HT as described on line 489? Should OlinkID be renamed as OlinkID_HT? Or should there be 3 separate OID columns?

Reply:
Table shows "MedianCenteredNPX" but text says that this is named "MedianAdjustedNPX" - which one is accurate?